model/schema/core.yaml

```diff
@@ -417,6 +436,15 @@ classes:
       - genome_assembly
       - maps_to_chromosome
 
+  AssemblyComponentDTO:
```

Comment:
@gildossantos Thanks for taking a look! I had a similar question this morning for @markquintontulloch regarding where a DQM should submit, for example, a UniProt ID as the primary ID for a protein when the MOD does not have its own curie/ID. We think submitting to the "mod_entity_id" field is fine for now, with the interpretation being that is the identifier uses for the object currently rather than necessarily a MOD's own identifier. If/when we mint Alliance AGRKB curies for this and other classes, we can move the, for example, UniProt ID into the cross references field and leave the mod_* fields blank. I agree we want to be careful not to make it look as if we're claiming that a MOD is claiming an external ID as their own